src/api/controllers/articleController.js

```diff
@@ -0,0 +1,68 @@
+import cloudinary from 'cloudinary';
+import model from '../models';
+import slugHelper from '../../helpers/slug.maker';
+import statuses from '../../helpers/constants/status.codes';
+
+const { Article } = model;
+
+/**
+ * this is the article controller
+ *
+ * @export
+ * @class ArticleController
+ */
+class ArticleController {
+  /**
+   * allow an author to update his/her article
+   *
+   * @author Frank Mutabazi
+   * @static
+   * @param {object} req the request
+   * @param {object} res the response to be sent
+   * @memberof ArticleController
+   * @returns {Object} res
+   */
+  static async updateArticle(req, res) {
+    const { slug } = req.params;
+    const {
+      title, description, body, tagList, category
+    } = req.body;
+
+    let { coverImage } = req.Existing;
+
+    const slugCreation = slugHelper(title || req.Existing.title);
+
+    if (req.file) {
+      const image = await cloudinary.v2.uploader.upload(req.file.path);
+      coverImage = image.secure_url;
+    }
+    const updateContent = {
```

Comment:
@MCFrank16 not sure why you need this `updateContent` object

Reply:
it is just there to organize properly what is going to be saved in the database